rosdep/python.yaml

```diff
@@ -3709,13 +3709,6 @@ python-requests:
       packages: [requests]
   ubuntu:
     '*': [python-requests]
-    trusty_python3: [python3-requests]
-    utopic_python3: [python3-requests]
-    vivid_python3: [python3-requests]
-    wily_python3: [python3-requests]
-    xenial_python3: [python3-requests]
-    yakkety_python3: [python3-requests]
-    zesty_python3: [python3-requests]
```

Comment:
I don't think we should remove these; it is possible that someone is depending on them, even though this method is deprecated.

Reply:
Yeah, we'd prefer to keep these there for compatibility in case someone is using them. We'll clean them out in the future when we do a systematic pruning of old distros.

Reply:
Of course, changed it back.

Just the new changes in there now.